# Request constants
int8 LAND=0
int8 TAKEOFF=1
int8 HOVER=2
int8 POSCTL=3
# Request fields
int8 set_state
---
bool status

